### ISSUE TITLE
RNN Transducer loss backward computation accuracy

### ISSUE THREAD

**Issue by HawkAaron** [awni/transducer#1]
Thanks for your great work, but there is still about 1e-3 error between numeric and backward gradient, 
is there any trick to be more accurate ?

**Comment by awni** [comment]
I haven't had any numerical issues when training with this loss function. It converges fine and the c implementation closely matches the reference python implementation which passes the numerical grad check to pretty high tolerance.  Have you seen numerical issues or convergence problems when using the loss function in practice?
